src/main/scala/viper/server/protocol/ViperIDEProtocol.scala

```diff
@@ -57,19 +58,19 @@ object ViperIDEProtocol extends akka.http.scaladsl.marshallers.sprayjson.SprayJs
 
   implicit val position_writer: RootJsonFormat[Position] = lift(new RootJsonWriter[Position] {
     override def write(obj: Position): JsValue = JsObject(
-        "file" -> (if (obj.file != null) {
-          //FIXME this hack is needed due to the following bug in Silver: https://bitbucket.org/viperproject/silver/issues/232
-          obj.file.toJson
-        } else {
-          JsString("<undefined>")
-        }),
-        "start" -> JsString(s"${obj.line}:${obj.column}"),
-        "end" -> (obj.end match {
-          case Some(end_pos) =>
-            JsString(s"${end_pos.line}:${end_pos.column}")
-          case _ =>
-            JsString(s"<undefined>")
-        }))
+      "file" -> (if (obj.file != null) {
+        //FIXME this hack is needed due to the following bug in Silver: https://bitbucket.org/viperproject/silver/issues/232
```

Comment:
Please update the comment with a link to 
https://github.com/viperproject/silver/issues/253